labs/Variables/readme.md

```diff
@@ -30,28 +30,28 @@ MacOS:
 From now on, read "Create a new project" as
 
  - Create a new project using the "Console Application" template for C\#.
- - Pick simple and valid names for your project and solution, and make sure you save it in the right place, like a folder that matches the lab name.
- - Compile and execute your program frequently, for instance upon completion of every lab task.
+ - Pick simple and valid names for your project and solution, and save it in a suitable place (ex. a folder with a name that matches the name of the lab).
+ - Compile and execute your program frequently (ex. upon completion of every lab task).
  - *Do not hesitate to change your program to answer questions; you're in a lab, you're supposed to conduct experiments!*
  - If you get an error when trying to compile your program, go to the line where your IDE thinks there is an error.
 
-And, _even if it is not explicitly asked_, you should save your work once you are done, then re-open it at the beginning of the next session to make sure you saved it properly.
+And, _even if it is not explicitly stated_, you should save your work once you are done and reopen it at the beginning of the next session to verify that you saved it properly.
 
 If you want a UCA or your instructor to check your answer to a previous lab, just ask!
 
 
-## First Variables Manipulation
+## First Variable Manipulations
 
 #. Create a new project as indicated above.
 #. In the `Main` method, add three statements:
 
-    - One that declares a `string` variable named `lastName`.
-    - One that declares a `string` variable named `firstName`.
+    - One that declares a `string` variable named `lastName`,
+    - One that declares a `string` variable named `firstName`, and
     - One that declares an `int` variable named `classOf`.
 
-#. Below those three statements, write statements that assign your last name to the first variable, your first name to the second variable, and your anticipated graduation year (i.e., 2025, for instance) to the third variable.
+#. Below those three statements, write statements that assign your last name to the first variable, your first name to the second variable, and your anticipated graduation year (ex. 2025) to the third variable.
```

Comment:
"i.e., " is what we use in math., but your "ex." is probably much better, @dpdiest . I'll have to make that substitution _in many places_. I can automate that, probably.